### PR TITLE
Mobile 3919

### PR DESCRIPTION
**Pull request by dpalou**

### CODE REVIEW

File: src/addons/mod/assign/pages/edit/edit.ts [hunk 1]
```diff
@@ -204,6 +215,44 @@ export class AddonModAssignEditPage implements OnInit, OnDestroy, CanLeave {
...
 
+    /**
+     * Start the submission if needed.
+     *
+     * @param submissionStatus Current submission status.
+     * @return Promise resolved with the new submission status if it changed, original submission status otherwise.
+     */
+    protected async startSubmissionIfNeeded(
+        submissionStatus: AddonModAssignGetSubmissionStatusWSResponse,
+        options: AddonModAssignSubmissionStatusOptions,
```
Comment (crazyserver) [inline src/addons/mod/assign/pages/edit/edit.ts:244]: Missing param on JS doc

File: src/core/services/utils/time.ts [hunk 2]
```diff
@@ -240,17 +187,21 @@ export class CoreTimeUtilsProvider {
...
-     * Returns duration in a short human readable format: minutes and seconds, in fromat: 3' 27''.
+     * Converts a number of seconds into a short human readable format: minutes and seconds, in fromat: 3' 27''.
      *
-     * @param duration Duration in seconds
-     * @return Duration in a short human readable format.
+     * @param seconds Seconds
+     * @return Short human readable text.
      */
-    formatDurationShort(duration: number): string {
+    formatTimeShort(duration: number): string {
```
Comment (crazyserver) [inline src/core/services/utils/time.ts:204]: Should this be moved to CoreTime singleton?

File: src/theme/theme.base.scss [hunk 3]
```diff
@@ -504,8 +504,21 @@ ion-toast {
     @include media-breakpoint-down(sm) {
         &::part(container) {
             flex-direction: column;
+            align-items: flex-end;
         }
     }
+
+    &.core-danger-toast {
```
Comment (crazyserver) [inline src/theme/theme.base.scss:511]: This can be automatized using a for each (like warning cards) this way all colors will be implemented

**Review by crazyserver — CHANGES_REQUESTED** [review]

File: src/core/services/utils/time.ts (hunk 2 above)
Comment (dpalou) [reply src/core/services/utils/time.ts:177]: Done, I added a new commit for this.

File: src/theme/theme.base.scss (hunk 3 above)
Comment (dpalou) [reply src/theme/theme.base.scss]: Right, I changed it.

**Review by crazyserver — APPROVED** [review]

**Pull request merged**